client/src/panels/panel_declarations/services/single_service_panels/service_standards.js

```diff
@@ -85,10 +97,76 @@ export class ServiceStandards extends React.Component {
           ),
       },
     };
+
+    const num_met = _.reduce(
+      data,
+      (accumulator, standard) => {
+        return standard.is_target_met ? accumulator + 1 : accumulator;
+      },
+      0
+    );
+
+    const large_icons = {
+      met: (
+        <IconCheck
+          key="met"
+          title={text_maker("standard_met")}
+          color={color_scale("standard_met")}
+          width={41}
+          vertical_align={"0em"}
+          alternate_color={false}
+          inline={false}
+        />
+      ),
+      not_met: (
+        <IconAttention
+          key="not_met"
+          title={text_maker("standard_not_met")}
+          color={color_scale("standard_not_met")}
+          width={41}
+          vertical_align={"0em"}
+          alternate_color={false}
+          inline={false}
+        />
+      ),
+    };
+
+    const filtered_data = _.filter(
+      data,
+      ({ is_target_met }) =>
+        !_.isEmpty(active_list) &&
+        (active_list.length === standard_statuses.length ||
+          is_target_met === _.includes(active_list, "met"))
+    );
+
     return (
       <Panel title={text_maker("service_standards_title")}>
         <TM className="medium_panel_text" k="service_standards_text" />
-        <DisplayTable data={data} column_configs={column_configs} />
+        <FilterTable
+          items={_.map(standard_statuses, (status_key) => ({
+            key: status_key,
+            count: status_key === "met" ? num_met : data.length - num_met,
```

Comment:
Just a super minor thing because I'm not a big fan of this ternary

How about making `num_met` to a function like `get_status_sum` where it takes an argument `is_status_met`, pass in `status_key==="met"`, and I think you should be able to compare `is_status_met` with `is_target_met` to calculate separate sum for "met" and "not_met"

it's really not much different but I think that's slightly more accurate way to do it without this ternary where you do `data.length - num_met`